packages/algob/src/lib/files.ts

```diff
@@ -4,11 +4,19 @@ import YAML from "yaml";
 
 import { BuilderError } from "../internal/core/errors";
 import { ERRORS } from "../internal/core/errors-list";
+import { ASSETS_DIR } from "../internal/core/project-structure";
+
+const msigExt = ".mlsig";
+const tealExt = ".teal";
 
 function normalizePaths (mainPath: string, paths: string[]): string[] {
   return paths.map(n => path.relative(mainPath, n));
 }
 
+function writeFile (filename: string, content: string): void {
+  fs.writeFileSync(filename, content);
+}
```

Comment:
this function is not needed, we can use `fs.writeFileSync` directly.

Reply:
removed